packages/server/lib/browsers/chrome.ts

```diff
@@ -324,7 +324,7 @@ const onReconnect = (client: CRIWrapper.Client) => {
   // if the client disconnects (e.g. due to a computer sleeping), update
   // the frame tree on reconnect in cases there were changes while
   // the client was disconnected
-  _updateFrameTree(client)()
+  return _updateFrameTree(client)()
```

Comment:
Is this asynchronous? Do we need to await it in `criClient.create`

Reply:
it is asynchronous, but we do not need to await it. The `return` statement here is to appease the linter. The waiting gets handled by the `_isAUTFrame` function itself [here](https://github.com/cypress-io/cypress/pull/21237/files#diff-e2888f195a1c64929f7e4de7cb14eba872a682eb1cd29f8d50b87131c1f78c1aR399).